src/polycubed/src/rest_server.cpp

```diff
@@ -150,6 +157,27 @@ void RestServer::shutdown() {
   } catch (const std::runtime_error &e) {
     logger->error("{0}", e.what());
   }
+  Rest::Resources::Endpoint::Resource::kill = true;
+  Rest::Resources::Endpoint::Resource::data_cond.notify_one();
+}
+
+void RestServer::load_last_topology() {
+  startup = true;
+  std::ifstream myFile(get_last_topology_path());
```

Comment:
Use more meaningful names. MyFile does not mean nothing.
topologyFile looks much better than myFile.